tests/installation/bootloader_s390.pm

```diff
@@ -94,13 +94,16 @@ ENTER
 ENTER
 EO_frickin_boot_parms
 
+    # Define memory to behave the same way as other archs
+    # and to have the same configuration through all s390 guests
+    # convert MB -> GB, it's not like perl can't calculate
+    my $ram = get_var('QEMURAM', 1024) / 1024;
```

Comment:
But what I realized just now is, that you can indeed do... I missed that yesterday somehow
`DEFINE STORAGE 1024M`
`STORAGE = 1G`

So i suggest to just remove the `/ 1024`